README.md

```diff
@@ -95,8 +95,8 @@ homedir() | home()
 cd() | cd()
 joinpath() | joinpath()
 basename() | basename()
-N/A | filename
-N/A | extension
+splitext(basename())[1] | filename
+splitext(basename())[2] | extension
```

Comment:
I wasn't really planning on including behaviour that required multiple method calls using base, but I guess it doesn't hurt to provide the mapping anyways.